custom-elements/throw-on-dynamic-markup-insertion-counter-construct.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+ <html>
+ <head>
+ <title>Custom Elements: create an element for a token must increment and decrement document's throw-on-dynamic-markup-insertion counter</title>
+ <meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+ <meta name="assert" content="Invoking document.open, document.write, document.writeln, and document.write must throw an exception when the HTML parser is creating a custom element for a token">
+ <meta name="help" content="https://html.spec.whatwg.org/multipage/parsing.html#create-an-element-for-the-token">
+ <meta name="help" content="https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#throw-on-dynamic-markup-insertion-counter">
+ <script src="/resources/testharness.js"></script>
+ <script src="/resources/testharnessreport.js"></script>
+ <script src="../../../../imported/w3c/web-platform-tests/custom-elements/resources/custom-elements-helpers.js"></script>
```

Comment:
This is probably an incorrect link and should be changed to "resources/custom-elements-helpers.js". Ditto below.
A nice way might be to verify the tests are running properly using wpt run or wpt serve directly in WPT folder.

If you do not want to have to do a review here, another way is to do the changes in WebKit repo (migrate these tests in LayoutTests/imported/w3c/web-platform-tests/custom-elements) and use Tools/Scripts/export-w3c-test-changes script to create the PR. The WPT PR will then be mergeable directly.

Reply:
As previously discussed on webkit-dev, that process is not acceptable. Please don't do that. We don't want any tests not haven't been exported to in `LayoutTests/imported/`.

Reply:
@rniwa I'm not sure I understand your concern? IIUC, @youennf is suggesting to

1) upload a patch moving the tests (using a new WebKit bug)
2) create a WPT PR (using the export script)
3) merge the WPT PR
4) land the patch into WebKit

So the tests are already in upstream WPT before being imported in WebKit.

Reply:
Sure, as long as the test change won't be committed into WebKit, that's fine.

Reply:
@youennf   I have corrected the file path